docs/tutorials/hello.rst

```diff
@@ -64,3 +64,9 @@ change the scroll method's argument.
     in a way that MicroPython can't understand. Check you're not missing any
     special characters like ``"`` or ``:``. It's like putting. a full stop in
     the middle of a sentence. It's hard to understand exactly what you mean.
+
+    Your microbit may stop responding: you cannot flash new code to it or
```

Comment:
I did yesterday, @ntoll saw it. The microbit stopped accepting new flashes after I held down the reset button for a few seconds when my program wasn't behaving as expected.

Perhaps we should add something to the tutorial about what exactly the reset button does and when you should push it.